app/lib/result_handler/result_handler.rb

```diff
@@ -0,0 +1,36 @@
+require 'matcher'
+module ResultHandler
+  include Matcher
+  def handle_successful(result, &block)
```

Comment:
Do not skimp on empty new lines, it's not the 90th years :D Lets make it readable while [skimming](https://en.wikipedia.org/wiki/Speed_reading#Skimming_and_scanning)